docs/edgeql/ddl/scalars.rst

```diff
@@ -48,6 +48,10 @@ subtype with constraints.
 :eql:synopsis:`EXTENDING <supertype>`
     Optional clause specifying the *supertype* of the new type.
 
+    The :eql:synopsis:`<supertype>` is an
+    :eql:type:`enumerated type <std::enum>` declaration, in which
+    case an enumerated scalar type is defined.
```

Comment:
Why are we saying that `<supertupe>` must be an enum declaration in a *generic* section about scalars?

Reply:
The wording of this paragraph is weird. Rephrase to something like this: 

To define an enumerated scalar type specify :eql:type:`enumerated type <std::enum>` declaration in :eql:synopsis:`<supertype>` .

Reply:
That sentence is missing a leading "if" and was intended like this:

"
    If the :eql:synopsis:`<supertype>` is an
    :eql:type:`enumerated type <std::enum>` declaration, an enumerated scalar type is defined.
"